lib/model/settings.dart

```diff
@@ -203,6 +203,57 @@ enum BoolGlobalSetting {
   };
 }
 
+/// An int-valued, account-independent setting the user might set.
+///
+/// These are recorded in the table [IntGlobalSettings].
+/// To read the value of one of these settings, use [GlobalSettingsStore.getInt];
+/// to set the value, use [GlobalSettingsStore.setInt].
+///
+/// To introduce a new setting, add a value to this enum.
+/// Avoid re-using any old names found in the "former settings" list.
+///
+/// To remove a setting, comment it out and move to the "former settings" list.
+/// Tracking the names of settings that formerly existed is important because
+/// they may still appear in users' databases, which means that if we were to
+/// accidentally reuse one for an unrelated new setting then users would
+/// unwittingly get those values applied to the new setting,
+/// which could cause very confusing buggy behavior.
+///
+/// (If the list of former settings gets long, we could do a migration to clear
+/// them from existing installs, and then drop the list.  We don't do that
+/// eagerly each time, to avoid creating a new schema version each time we
+/// finish an experimental feature.)
+enum IntGlobalSetting {
+  /// A non-setting to ensure this enum has at least one value.
+  ///
+  /// Leave this in place even when there are other settings too.
+  /// That way when we remove those, this is already here.
+  /// (Having one stable value in this enum is also handy for tests.)
+  placeholderIgnore,
```

Comment:
```suggestion
  /// A non-setting to ensure this enum has at least one value.
  ///
  /// (This is also handy to use in tests.)
  placeholderIgnore,
```

The other text doesn't make so much sense here as it does on BoolGlobalSetting, because the actual setting in this enum isn't experimental and so isn't something we particularly intend to ever remove.